src/dashboard/Push/PushNew.react.js

```diff
@@ -670,21 +714,40 @@ export default class PushNew extends DashboardView {
       {this.renderExperimentContent(fields, setField)}
     </Fieldset> : null;
 
+    const richmediaFieldsLegend = PARSE_SERVER_SUPPORTS_PUSH_RICH_MEDIA ?
+      'Rich media push notification' :
+      'Rich notifications';
+
+    const richmediaFieldsDescription = PARSE_SERVER_SUPPORTS_PUSH_RICH_MEDIA ?
+      'We can send images and videos directly to your app.' :
+      "If your push hasn't been send with rich media, it won't get setup by your dev.";
```

Comment:
Using double quotes here but single everywhere else.

Also, should probably be something like: `Rich media is not currently supported.`